<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTableWidget</class>
 <widget class="QWidget" name="MainTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainTableWidget</string>
  </property>
  <widget class="QWidget" name="status_show" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-49</y>
     <width>531</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1191d3;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(&quot;:/res/ring.png&quot;)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ffffff</string>
      </property>
      <property name="text">
       <string>MSG</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>49</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="table_show">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#000000;</string>
       </property>
       <property name="text">
        <string>今日的课程已上完</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="class_show_widget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="class_time">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>--:--:--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="hide_window">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-2</y>
     <width>21</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(200,200,200,0.5);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="timer_show" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>181</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1191d3;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(&quot;:/res/ring.png&quot;)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#fff</string>
      </property>
      <property name="text">
       <string>NO_TIMER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>hide_window</zorder>
  <zorder>status_show</zorder>
  <zorder>timer_show</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
